app/src/main/java/org/simple/clinic/contactpatient/ContactPatientEffectHandler.kt

```diff
@@ -46,9 +46,19 @@ class ContactPatientEffectHandler @AssistedInject constructor(
         .addTransformer(CancelAppointment::class.java, cancelAppointment(schedulers.io()))
         .addTransformer(MarkPatientAsMovedToPrivate::class.java, markPatientAsMovedToPrivate(schedulers.io()))
         .addTransformer(RemoveAppointment::class.java, removeAppointment(schedulers.io()))
+        .addTransformer(MarkPatientAsTransferredToAnotherFacility::class.java, markPatientAsMovedToPublic(schedulers.io()))
         .build()
   }
 
+  private fun markPatientAsMovedToPublic(scheduler: Scheduler): ObservableTransformer<MarkPatientAsTransferredToAnotherFacility, ContactPatientEvent> {
+    return ObservableTransformer { effects ->
+      effects
+          .observeOn(scheduler)
+          .doOnNext { patientRepository.updatePatientStatusToMigrated(patientUuid = it.patientUuid) }
+          .map { PatientMarkAsMigrated }
```

Comment:
The `RemoveAppointment` effect which is triggered by this event will end up hard-coding the appointment cancel reason to `MovedToPrivatePractioner` which is the wrong reason here.

Reply:
Got it. Removing the `RemoveAppointment` effect and continue using the `CancelAppointment` effect